src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -21,6 +21,7 @@
     <ImportVSSDKTargets>true</ImportVSSDKTargets>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <CopyNuGetImplementations>true</CopyNuGetImplementations>
+    <DeployExtension Condition="'$(VisualStudioVersion)' == '14.0'">false</DeployExtension>
```

Comment:
Okay.  I've gotten bitten by that in the past.  Maybe it's an issue that's since been fixed.